src/core/highlight.js

```diff
@@ -48,18 +48,22 @@ export async function run(conf, doc, cb) {
     .from(
       doc.querySelectorAll("pre:not(.idl):not(.nohighlight),code.highlight")
     )
+    .map(element => {
+      element.setAttribute("aria-busy", "true");
+      return element;
+    })
     .map(element => {
       return new Promise((resolve, reject) => {
         if (element.textContent.trim() === "") {
-          return resolve(); // no work to do
+          return resolve(element); // no work to do
         }
         const msg = {
           action: "highlight",
           code: element.textContent,
           id: Math.random().toString(),
           languages: getLanguageHint(element.classList),
         };
-
+        element.setAttribute("aria-live", "polite");
```

Comment:
This presents a significant accessibility problem for screen reader users trying to read a spec containing a lot of code examples, because every code sample is sent to the the screen reader buffer to announce at its earliest convenience. I've sometimes been unable to test a page with VoiceOver with a W3C spec open in another tab, because all I hear are source code examples from the spec being announced.